omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -94,6 +94,33 @@
     <div class="col-md-12 col-sm-12 col-xs-12">
         <form method="get" id="patient-search-form" onsubmit="return false">
             <input class="form-control input-sm input-lg" type="text" id="patient-search" placeholder="${ ui.message("coreapps.findPatient.search.placeholder") }" autocomplete="off" <% if (doInitialSearch) { %>value="${doInitialSearch}"<% } %>/><i id="patient-search-clear-button" class="small icon-remove-sign"></i>
+            <% if(patientSearchExtensions){
+			
+            <select id="patient-gender-search">
```

Comment:
These are the lines to which the error points. @k-joseph .

Reply:
Hey @Irenyak1  I solved this problem. but I'm getting unexpected travis CI build failure. On my system, I can able to build this module without any problem. Suggest me any solution